ntc_templates/templates/fsas_sir_show_ether.textfsm

```diff
@@ -0,0 +1,30 @@
+Value Required ETHER_GROUP (\d+)
+Value Required ETHER_PORT (\d+)
+Value DESCRIPTION (\S+)
+Value STATUS (.+?)
+Value STATUS_AUTO (auto)
+Value STATUS_SPEED ([\d]+[MG])
+Value STATUS_DUPLEX (Full|Half)
+Value STATUS_MDI (MDI(-X)?)
+Value MEDIA (\S+)
+Value FLOW_CONTROL_SEND (\w+)
+Value FLOW_CONTROL_RECEIVE (\w+)
+Value TYPE (.+?)
+Value SINCE (.+?)
+Value CONFIG_MODE (\S+)
+Value CONFIG_MDI (\S+)
+Value CONFIG_MEDIA (\S+)
+
+Start
+  ^\[ETHER -> Continue.Record
+  ^\[ETHER\s+GROUP\-${ETHER_GROUP}\s+PORT\-${ETHER_PORT}\]\s*$$
+  ^description\s+:(\s+${DESCRIPTION})?\s*$$
+  ^status\s+:\s+(${STATUS_AUTO}\s+)?${STATUS_SPEED}\s+${STATUS_DUPLEX}\s+${STATUS_MDI}\s*$$ -> Continue
```

Comment:
Change to use just the STATUS capture group.
Remove the Continue directive which in this case causes parsing issues when the regex for STATUS is changed.

```suggestion
  ^status\s+:\s+(${STATUS}\s+)?${STATUS_SPEED}\s+${STATUS_DUPLEX}\s+${STATUS_MDI}\s*$$
```

Reply:
It's possible I should have just worked on this locally in order to easily regenerate the yaml structured output.
I may convert all of these suggestions into a PR if @caribouHY hasn't accepted them by that point.
Thank you.